modules/apps/callbacks/callbacks_test.go

```diff
@@ -289,3 +291,31 @@ func (s *CallbacksTestSuite) AssertHasExecutedExpectedCallbackWithFee(
 	}
 	s.AssertHasExecutedExpectedCallback(callbackType, isSuccessful)
 }
+
+// GetExpectedEvent returns the expected event for a callback.
+func GetExpectedEvent(
+	packetDataUnmarshaler porttypes.PacketDataUnmarshaler, remainingGas uint64, data []byte, srcPortID,
+	eventPortID, eventChannelID string, seq uint64, callbackType types.CallbackType, expError error,
+) (abci.Event, bool) {
+	var (
+		callbackData types.CallbackData
+		err          error
+	)
+	switch callbackType {
+	case types.CallbackTypeReceivePacket:
```

Comment:
if the switch has a single case and a default should we just use an `if else`?

Reply:
done